packages/charts/src/charts/line/line-chart.component.tsx

```diff
@@ -121,6 +134,9 @@ export const LineChart: FC<Props> = ({
     >
       <Grid xScale={xScale} yScale={yScale} />
       <Axes xScale={xScale} yScale={yScale} />
+      {(xAxisTitle || yAxisTitle) && (
```

Comment:
Please, refactor it to component with single responsibility that will no handle both cases - but will receive a proper configuration. 
```
{xAxisTitle && (
   <Title
        axisTitle={title}
        orientation={Orientation.HORIZONTAL}
        {...typography}
      />
)}
```